cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -128,13 +153,13 @@ public void tryConnect(ConnectionConfig connectionConfig, PrintStream output, bo
   }
 
   public void updateAccessToken(PrintStream output) throws IOException {
-    AccessToken newAccessToken = getNewAccessToken(clientConfig.getConnectionConfig(), output, false);
+    AccessToken newAccessToken = getNewAccessToken(clientConfig, output, false);
     clientConfig.setAccessToken(newAccessToken);
   }
 
   private void checkConnection(ClientConfig baseClientConfig,
-                               ConnectionConfig connectionInfo,
```

Comment:
Revert changes to this method. `connectionInfo` is intended to override the `baseClientConfig`'s connection info, where `connectionInfo` is intended to be the new connection we want to check whereas `baseClientConfig` has timeout config + current connection.